Shared/Samples/Show popup/README.md

```diff
@@ -0,0 +1,34 @@
+# Show popup
+
+Show a predefined popup from a web map.
+
+![Show popup screenshot](show-popup.png)
```

Comment:
We may want to downsize this image a bit. It's smaller in the other READMEs and overpowers the rest of the content a bit. I _think_ the others use a width of 276.